### PR TITLE
Switch to gzip compression in QR codes

### PR DESCRIPTION
**Pull request by dhruvbansal**
Switch to using gzip instead of LZMA when (de)compressing data from/to QR codes.

### CODE REVIEW

File: hermit/qrcode/format.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
 
 from hermit.errors import InvalidSignatureRequest
 
 
 def decode_qr_code_data(encoded: bytes) -> str:
+    if not isinstance(encoded, (bytes,)):
+        raise InvalidSignatureRequest("Can only decode bytes")
+    if encoded == b'':
+        raise InvalidSignatureRequest("Cannot encode empty bytes")
```
Comment (destrys) [inline hermit/qrcode/format.py]: should this say 'can't *decode* empty bytes'?

File: tests/fixtures/large_bitcoin_signature_request.json [hunk 2]
```diff
@@ -1,306 +0,0 @@
-{
```
Comment (destrys) [inline tests/fixtures/large_bitcoin_signature_request.json]: why is this deleted in full? I forget why it's here.

is there a corresponding png to delete?

Comment (dhruvbansal) [reply tests/fixtures/large_bitcoin_signature_request.json]: The naming was confusing.  I've renamed this fixture to `too_large_bitcoin_signature_request.json` which makes it's usage more clear.  Since it's too large, there is no corresponding image.
